js/src/forum/fragments/PostQuoteButton.js

```diff
@@ -0,0 +1,55 @@
+import Fragment from 'flarum/Fragment';
+import icon from 'flarum/helpers/icon';
+
+import reply from '../utils/reply';
+
+export default class PostQuoteButton extends Fragment {
+  constructor(post) {
+    super();
```

Comment:
Doesn't seem to be needed.

Reply:
Errors without it. https://stackoverflow.com/a/31079103